rust/lon/src/cli.rs

```diff
@@ -529,6 +564,11 @@ impl fmt::Display for CommitMessage {
                 writeln!(&mut commit_message, "• {name}:")?;
                 writeln!(&mut commit_message, "    {}", summary.old_revision)?;
                 writeln!(&mut commit_message, "  → {}", summary.new_revision)?;
+
+                if let Some(details) = &summary.details {
+                    writeln!(&mut commit_message)?;
+                    write!(&mut commit_message, "{details}")?;
+                }
```

Comment:
We can call this just once after the enclosing if else statement.

Reply:
Hmmm, not really, the rev_list is attached to the `summary`, not the commit message directly, which can have multiple summaries

Reply:
Ah I see. Ok but then we need to make sure that the indentation is correct  so that it actually appears under the bullet point